rero_ils/modules/holdings/jsonschemas/holdings/holding-v0.0.1.json

```diff
@@ -151,6 +153,97 @@
             }
           }
         },
+        "frequency": {
+          "title": "Publication frequency of the issue",
+          "description": "Used for a publishing pattern that has a fundamental periodicity.",
+          "type": "string",
+          "enum": [
+            "Annual",
+            "Monthly",
+            "Daily",
+            "Quarterly",
+            "Weekly",
+            "Bimonthly",
+            "Biweekly",
+            "Semimonthly",
+            "Semiannual",
+            "Semiweekly",
+            "Three times a year",
+            "Three times a month",
+            "Three times a week",
+            "Biennial",
+            "Triennial",
+            "Completely irregular"
```

Comment:
Either these should be codes, either we do not need the form options labels and values below.

Reply:
The `form` is removed